tests/ha/ctdb.pm

```diff
@@ -0,0 +1,121 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test ctdb resource agent
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+use utils qw(systemctl exec_and_insert_password);
```

Comment:
`exec_and_insert_password` seems to not be used.

Reply:
Good catch, I used it in a first version and forgot to remove it.